tools/CustomTasks/Utils.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace CustomTasks
+{
+    class Utils
+    {
+        //The XmlWriter can't handle &#xE0E5 unless we escape/unescape the ampersand
+        public static string UnEscapeAmpersand(string s)
```

Comment:
Why is that value special?

Reply:
& is special in XML.
XAML framework expects setting like this: Glyph="&#xE0E3;"
when xml parse it, "&#xE0E3;", it's been treat as one Unicode. 
but when we save it, it becomes things like Glyph="". 
When we pass it to XAML framework, it becomes another unknown character.